### PR TITLE
lact: Add at v0.5.4

### PR DESCRIPTION
**Pull request by uni-dos**
**Summary**
Add the lact package, which allows users of AMD cards to undervolt their gpus.

**Test Plan**

Run lact in virtual machine to see if it runs.

**Checklist**

- [x] Package was built and tested against unstable

I am not able to properly test the package because I am on stable.

Resolves https://github.com/getsolus/packages/issues/2674

### CODE REVIEW

File: packages/l/lact/package.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+homepage   : https://github.com/ilya-zlobintsev/LACT
+license    : MIT
+component  : system.utils
+summary    : Linux AMDGPU Configuration Tool 
+networking : yes
+description: |
+    This application allows you to control your AMD GPU on a Linux system.
+builddeps  :
+    - pkgconfig(gtk4)
+    - pkgconfig(hwdata)
```
Comment (algent-al) [inline packages/l/lact/package.yml:15]: No need to add `pkgconfig(hwdata)`  to builddeps because it is part of `system.base` component.

**Review by algent-al — DISMISSED** [review]
Try to build without `pkgconfig(hwdata)`.

File: packages/l/lact/package.yml (hunk 1 above)
Comment (uni-dos) [reply packages/l/lact/package.yml:15]: I see. I did not know that. I've rebuilt it without hwdata.

**Comment by malfisya** [comment]
Package request has been approved

**Comment by ReillyBrogan** [comment]
I made a few quick changes:
- Updated it to the latest
- Added monitoring.yml
- Fixed the systemd service file ordering
- Made it so that the systemd service is automatically started
- Switched it so that it uses /run paths for the sockets instead of /var/run

**Review by ReillyBrogan — APPROVED** [review]

**Pull request merged**